server/src/test/java/org/opensearch/cluster/service/MasterServiceTests.java

```diff
@@ -691,6 +691,9 @@ public void clusterStateProcessed(String source, ClusterState oldState, ClusterS
                     submittedTasksPerThread.get(entry.getKey()).get()
                 );
             }
+            // verify stats values after state is published
+            assertEquals(1, clusterManagerService.getStateStats().getUpdateSuccess());
+            assertEquals(0, clusterManagerService.getStateStats().getUpdateFailed());
```

Comment:
UT for failed metric

Reply:
CleanupFailure and failed update stats will be asserted properly in different tests.